### ISSUE TITLE
Setting "Customizable Switches" in webconf fails with Internal Error 500

### ISSUE THREAD

**Issue by riban-bw** [zynthian/zynthian-issue-tracking#494]
**Describe the bug**
If I set the first Customizable Switch to CC then change the second Customizable Switch I get Internal Error 500.

**To Reproduce**
1. Go to webconf Hardware/wiring
2. Change values of Customizable Switches 1,2

**Expected behaviour**
Configuration is applied.

**Actual behaviour**
Internal web server error 500.

**Setup:**
* HARDWARE
Raspberry Pi 4 Model B Rev 1.4
Soundcard: HifiBerry DAC+ ADC PRO
Display: ZynScreen 3.5 (v1)
Wiring: MCP23017_ZynScreen
GPIO Expander: MCP23017
* SYSTEM
Raspbian GNU/Linux 10 (buster)
Build Date: 2020-09-05
Memory: 10% (197M/1872M)
SD Card: 49% (14G/29G)
Temperature: 49.6ºC
* MIDI
Profile: default.sh
Fine Tuning: 440 Hz
Master Channel: 16
Single Active Channel: off
ZS3 SubSnapShots: on
* SOFTWARE
zyncoder: testing (7b58d8c)
zynthian-ui: sfizz (6c7912d)
zynthian-sys: sfizz (18d2a12)
zynthian-data: testing (f6a7b36)
zynthian-webconf: testing (38d33ed)

**Additional context**
Log
```
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]: Traceback (most recent call last):
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]:   File "/usr/local/lib/python3.7/dist-packages/tornado/web.py", line 1346, in _execute
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]:     result = method(*self.path_args, **self.path_kwargs)
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]:   File "/usr/local/lib/python3.7/dist-packages/tornado/web.py", line 2626, in wrapper
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]:     return method(self, *args, **kwargs)
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]:   File "/home/pi/zynthian-webconf/lib/wiring_config_handler.py", line 495, in post
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]:     self.get(errors)
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]:   File "/usr/local/lib/python3.7/dist-packages/tornado/web.py", line 2626, in wrapper
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]:     return method(self, *args, **kwargs)
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]:   File "/home/pi/zynthian-webconf/lib/wiring_config_handler.py", line 479, in get
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]:     super().get("Wiring", config, errors)
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]:   File "/usr/local/lib/python3.7/dist-packages/tornado/web.py", line 2626, in wrapper
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]:     return method(self, *args, **kwargs)
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]:   File "/home/pi/zynthian-webconf/lib/zynthian_config_handler.py", line 173, in get
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]:     super().get("config_block.html", title, config, errors)
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]:   File "/usr/local/lib/python3.7/dist-packages/tornado/web.py", line 2626, in wrapper
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]:     return method(self, *args, **kwargs)
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]:   File "/home/pi/zynthian-webconf/lib/zynthian_config_handler.py", line 121, in get
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]:     self.render("config.html", body=body, config=config, title=title, errors=errors)
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]:   File "/home/pi/zynthian-webconf/lib/zynthian_config_handler.py", line 97, in render
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]:     info['scrollTop'] = int(self.get_argument('_scrollTop', '0'))
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]: ValueError: invalid literal for int() with base 10: '1010.6666870117188'
Aug 29 08:12:16 zynthian1 zynthian_webconf.sh[427]: ERROR:tornado.access:500 POST /hw-wiring (192.168.1.107) 235.10ms
```

**Comment by riban-bw** [comment]
This is not always reproducible but when it goes wrong (which is quite often - @wyleu has mentioned it) it can generally be reproduced by changing switch 2 when switch 1 is configured. A work around seems to be to set switch 1 to None, configure switch 2 then to configure switch 1.

**Comment by riban-bw** [comment]
This is resolved by pull request https://github.com/zynthian/zynthian-webconf/pull/169.

**Comment by jofemodo** [comment]
I can't reproduce the error. Are you using Firefox or Chrome?
Anyway, as the fix seems quite unharmful, it's merged now ;-)

**Comment by riban-bw** [comment]
I could reproduce the issue quite often. I diagnosed with the debugger that the value returned was a string representation of a float. I tried to cast that to an int in a python shell and got the same error so it seems that under some conditions the scroll point results in a float being calculated and returned. This is definitely a fix for this issue. It may occur else where in the code - we should review, e.g. search for casting a string to an int.
